source/guides/administration.rst

```diff
@@ -0,0 +1,232 @@
+* :doc:`Managing Team and Channel Members </manage/cloud-team-and-channel>` - Manage synchronization, moderation, and membership settings.
+
+Workspace Compliance
+^^^^^^^^^^^^^^^^^^^^
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    Compliance Export </comply/cloud-compliance-export>
+    Data Retention Policy </comply/cloud-data-retention-policy>
+    Custom Terms of Service </comply/cloud-custom-terms-of-service>
+
+* :doc:`Compliance Export </comply/cloud-compliance-export>` - Create compliance reports from the Mattermost System Console. 
+* :doc:`Data Retention Policy </comply/cloud-data-retention-policy>` - Set custom data retention policies to manage how long Mattermost retains messages and file uploads.
+* :doc:`Custom Terms of Service </comply/cloud-custom-terms-of-service>` - Define custom terms of service for users to accept before joining your Mattermost server.
+
+Self-Hosted Admin
+-----------------
+
+This section of the guide is for administrators of self-hosted Mattermost servers. If you are the administrator for a Mattermost cloud workspace, please refer to the cloud section of this page.
+
+Self-Hosted Setup & Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Initial Setup
+"""""""""""""
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    Configuration Overview </configure/configuration-settings>
+    Custom Branding Tools </configure/custom-branding-tools>
+    Customize Mattermost </configure/customizing-mattermost>
+    SMTP Email Setup </configure/smtp-email>
+
+* :doc:`Configuration Overview </configure/configuration-settings>` - Setup and configure your Mattermost cloud workspace.
+* :doc:`Custom Branding Tools </configure/custom-branding-tools>` - Change Mattermost branding, site name, and description. 
+* :doc:`Customize Mattermost </configure/customizing-mattermost>` - Whitelabel the Mattermost server and apps.
+* :doc:`SMTP Email Setup </configure/smtp-email>` - Connect to an email server to send emails for password resets and system notifications.
+
+Advanced Configurations
+"""""""""""""""""""""""
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    Shared Channels </onboard/shared-channels>
+    Store Mattermost Configuration in the Database </configure/configuation-in-mattermost-database>
+    Email Templates </configure/email-templates>
+    Configuring CloudFront to host Mattermost static assets </configure/configuring-cloudfront-to-host-mattermost-static-assets>
+    Use an Outbound Proxy </configure/using-outbound-proxy>
+    Chinese, Japanese, and Korean Search </configure/enabling-chinese-japanese-korean-search>
+
+* :doc:`Shared Channels </onboard/shared-channels>` - Connect channels from multiple Mattermost servers in a federated architecture.
+* :doc:`Include Configuration in the Mattermost Database </configure/configuation-in-mattermost-database>` - Store Mattermost configuration information in your database rather than as a JSON file. We recommend this for high-availability environments.
```

Comment:
```suggestion
* :doc:`Include Configuration in the Mattermost Database </configure/configuation-in-mattermost-database>` - Store Mattermost configuration information in your database rather than as a JSON file. We recommend this for High Availability environments.
```